use aoc::y2020::operation_order_18::do_solve;
use std::io::Error;

fn main() -> Result<(), Error> {
    aoc::with_input(2020, 18, do_solve)
}
